docs/installation/linux.md

```diff
@@ -339,6 +339,33 @@ and is a single binary.
 
 ## macOS 10.12
 
+### Installation via Homebrew (preferred) - macOS 10.12
+
+[Homebrew][brew] is the missing package manager for macOS.
+If the `brew` command is not found,
+you need to install Homebrew following [their instructions][brew].
+
+Once you've installed Homebrew, installing PowerShell is easy.
+Just install [Homebrew-Cask][cask] and then you can install PowerShell:
+
+```sh
+brew tap caskroom/cask
```

Comment:
This line is installation instruction for cask.
It's not clear from the comment that these two lines correspond to two steps above.

I'd break it down into two parts.